z.zed

```diff
@@ -0,0 +1,463 @@
+definition notifications/integration {
```

Comment:
Everything else looks good to me, but was this `z.zed` file meant to be committed?

Reply:
no :( 
It was just a test, removing it now haha